dnf5/context.cpp

```diff
@@ -165,7 +165,10 @@ class Context::Impl {
 
     void print_info(std::string_view msg) const;
 
-    void set_output_stream(std::ostream & new_output_stream) { output_stream = new_output_stream; }
+    void set_output_stream(std::ostream & new_output_stream) {
```

Comment:
It might be more useful to have setters for both output and error in case users want to inspect them separately.

Reply:
Done, there is now also `Context::set_error_stream`.